app/views/fields/radiobutton-options.test.js

```diff
@@ -133,4 +133,39 @@ describe('radiobuttonOptions', () => {
         done();
       });
   });
+
+  it('should render the radiobutton option as an error if the context provided contains an error', (done) => {
+    const context = {
+      question: {
+        id: 'fieldId',
+        mainAdvice: 'Some really important main advice',
+        additionalAdvice: 'Some not so important additional advice',
+        options: [
+          {
+            value: 'first-option',
+            text: 'First Option',
+          },
+          {
+            value: 'second-option',
+            text: 'Second Option',
+          },
+        ],
+        error: {
+          message: 'Some error message',
+        },
+      },
+    };
+
+    const dummyApp = createDummyApp(context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        expect($('.nhsuk-error-message').text().trim()).toEqual('Error: Some error message');
+        expect($('#fieldId-error').length).toEqual(1);
```

Comment:
where does `#fieldId-error` come from and is there any particular reason you are checking that this exists?

Reply:
This is generated by the NHS component. It takes the element id and appends '-error'.

After discussion this could cause problems later down the line if the NHS component library is ever updated. We are probably best introducing our own div to check the rendered error state.